config/chromium-flags.conf

```diff
@@ -1,3 +1,4 @@
 --ozone-platform=wayland
 --ozone-platform-hint=wayland
 --enable-features=TouchpadOverscrollHistoryNavigation
+--load-extension=$HOME/.local/share/omarchy/applications/chromium/copy-url
```

Comment:
i wonder if we could use "~/.local/share/omarchy/applications/chromium/copy-url" and then could totally avoid the $HOME changes in this PR

Reply:
@hjanuschka nice! This works.

I had tried $HOME which doesn't work. But ~ works fine, great suggestion. I dropped the change to the update script.

Reply:
it works for me! :) but i had to change the shortcut, as my customized conf + german macbook keyboard

Reply:
Cool, thanks for testing. I added a little explanation to the commit to make it clear you can change it.